zh-Hans/site.json

```diff
@@ -4,8 +4,8 @@
   "navigation": {
     "TC39": {
       "title": "TC39",
-      "agenda": "下一次会议的议程",
-      "notes": "历史会议记录",
+      "agenda": "Meeting Agendas",
+      "notes": "Meeting Notes",
```

Comment:
i think the translations need to be in "not english"

Reply:
yes.  see `Issues` above

Reply:
ah. any reason not to just run them through google translate?

Reply:
> ah. any reason not to just run them through google translate?

https://github.com/tc39/tc39.github.io/pull/352#issuecomment-1793435099